src/tests/validity/variable_constant_condition.ab

```diff
@@ -0,0 +1,22 @@
+// Output
+// Outer: 42
+// If: 12
+// Else: 24
+
+main {
+    const x = 42
+    echo "Outer: {x}"
+
+    if x > 12 {
+        const x = 12
+        echo "If: {x}"
+    }
+
+    if x < 12 {
+        const x = 12
+        echo "If: {x}"
+    } else {
+        const x = 24
+        echo "Else: {x}"
+    }
+}
```

Comment:
Would you mind configuring your IDE or text editor to automatically terminate the last line of a file?

Reply:
I was editing this in github codespace which is probably why it happened. I have my editor configured to automatically terminate last line.